Task 3/infix-to-postfix.c

```diff
@@ -0,0 +1,114 @@
+#include<stdio.h>
```

Comment:
Yes! Better. Now just one final thing. 

You'll have to put this in your own folder.

So your full path should be something like:

`Task 3/InfixToPostfix/infix-to-postfix.c`

Can you add a multiline comment on the top of the program to show what the program does?

Reply:
You can also add the output screenshot in your own folder! 😄

Reply:
how to make a folder...please guide me

Reply:
If you are committing via the web app, you have to enter a slash ('/') in the filename part.

But fyi, this isn't a recommended way. You need to use something known as a `git client`. Find it here:

-  https://git-scm.com/